docs/conf.py

```diff
@@ -5,7 +5,7 @@
 https://www.sphinx-doc.org/en/master/usage/configuration.html
 
 Built docs locally with:
-sphinx-build -b html docs/source/ docs/build/html
+sphinx-build -b html docs docs/build/html
```

Comment:
That is a nice catch, I must have just copied the code and did not actually build the docs locally.